template/javascript/base/README.md

```diff
@@ -0,0 +1,24 @@
+# base
+
+## Project setup
+```
+yarn install
```

Comment:
could be selected package manager here (`yarn` - not `yarn install`, or `npm install` or `pnpm install`)